ms2deepscore/plotting.py

```diff
@@ -44,15 +44,18 @@ def plot_histograms(histograms, y_score_bins, bin_content=None,
     alpha = 1.0 #0.5
 
     # Create plot
-    plt.figure(figsize=(10,10))
+    plt.figure(figsize=(10, 10))
 
+    # Loop over each bin.
     for i in range(0, len(histograms)):
         data = histograms[len(histograms)-i-1][0]
-        data = data/max(data)
-        plt.fill_between(histograms[0][1][:100], -shift*i, [(-shift*i + x) for x in data], color=cmap1(i/10), alpha=alpha)
+        # Normalize the data to have the same area under the curve
+        data = data/sum(data)*len(data)/4
+        plt.fill_between(histograms[0][1][:len(data)], -shift*i, [(-shift*i + x) for x in data], color=cmap1(i/10), alpha=alpha)
```

Comment:
Here the plots are changed to not plot a graph normalized to the highest point, but instead to be normalized to the area under the curve.